features/steps/basic_replication.py

```diff
@@ -33,6 +33,46 @@ def add_table(context, table_name, pg_name):
         assert False, "Error creating table {0} on {1}: {2}".format(table_name, pg_name, e)
 
 
+@step('I pause wal replay on {pg_name:w}')
+def pause_wal_replay(context, pg_name):
+    # pause the wal replay process
+    try:
+       version = context.pctl.query(pg_name, "select pg_catalog.pg_read_file('PG_VERSION', 0, 2)").fetchone()
+       wal = version and version[0] and int(version[0].split('.')[0]) < 10 and "xlog" or "wal"
+       context.pctl.query(pg_name, "SELECT pg_{0}_replay_pause()".format(wal))
+    except pg.Error as e:
+        assert False, "Error pausing wal recovery on {0}: {1}".format(pg_name, e)
+
+
+@step('I resume wal replay on {pg_name:w}')
+def resume_wal_replay(context, pg_name):
+    # resume the wal replay process
+    try:
+       version = context.pctl.query(pg_name, "select pg_catalog.pg_read_file('PG_VERSION', 0, 2)").fetchone()
+       wal = version and version[0] and int(version[0].split('.')[0]) < 10 and "xlog" or "wal"
+       context.pctl.query(pg_name, "SELECT pg_{0}_replay_resume()".format(wal))
+    except pg.Error as e:
+        assert False, "Error resuming wal recovery on {0}: {1}".format(pg_name, e)
```

Comment:
It could be just a single function:
```python
@step('I {action:w} wal replay on {pg_name:w}')
def toggle_wal_replay(context, action, pg_name):
```

Reply:
Fixed